resources/views/create-word.blade.php

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/app.css">
+</head>
+<body>
+<!-- TODO: user should be able navigate to this page from the create word shortcut/button, and the back button needs to keep track of where we came from. its set to home for now.-->
+<h1 style="padding: 20px">
+    <a href="http://localhost:8080"><button class = "back-button">Back</button></a>
```

Comment:
ran into this bug while deploying the website, forgot to mention it - you can just provide a url like `/blabla` and it'll automatically insert the root website (in this case, `localhost:8080/blabla`). this makes it way easier to develop the site both locally (on `localhost:8080`) and on the server (`http://lucy.janissary.xyz`)
```suggestion
    <a href="/"><button class = "back-button">Back</button></a>
```